tests/test_config.py

```diff
@@ -12,13 +12,14 @@ def test_initial_values(config):
     """
     Test if the initial values of the Config class attributes are set correctly.
     """
-    assert config.debug_mode == False
-    assert config.continuous_mode == False
-    assert config.speak_mode == False
```

Comment:
Violation of PEP8 E712

<img width="663" alt="CleanShot 2023-04-30 at 22 12 38@2x" src="https://user-images.githubusercontent.com/52438637/235357720-aefccd9d-bcc0-4b85-b0b6-76eff7fdd398.png">